test/patch_test.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:sass_module_migrator/src/patch.dart';
+import 'package:source_span/source_span.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test("single patch applied", () {
+    var file = SourceFile.fromString("abcde");
+    var patch = Patch(file.span(2, 4), "fgh");
+    expect(Patch.applyAll(file, [patch]), equals("abfghe"));
+  });
```

Comment:
Nit: if you add empty lines between these tests, it'll help the file feel less cramped.